src/parser/monk/mistweaver/modules/spells/Vivify.js

```diff
@@ -27,20 +27,32 @@ class Vivify extends Analyzer {
   gustsHealing = 0;
   lastCastTarget = null;
   remDuringManaTea = 0;
+  countForGusts = false;
+  numberToCount = 0;
+
 
   on_byPlayer_cast(event) {
     const spellId = event.ability.guid;
+
     if (SPELLS.VIVIFY.id !== spellId) {
       return;
     }
+    if (this.combatants.players[event.targetID]) {
+      if (this.combatants.players[event.targetID].hasBuff(SPELLS.ESSENCE_FONT_BUFF.id, event.timestamp, 0, 0) === true) {
+        this.numberToCount++;
```

Comment:
Another style item, but we have chosen to use `+= 1` to increment. So just rework this is `this.numberToCount += 1;`

Reply:
Will do Mr.BossMan